### ISSUE TITLE
missing `AWS::CloudFront::MonitoringSubscription` resource type

### ISSUE THREAD

**Issue by hoonoh** [aws-cloudformation/cfn-lint#2476]
### CloudFormation Lint Version

0.71.0

### What operating system are you using?

Ubuntu

### Describe the bug

Hello, i noticed resource type `AWS::CloudFront::MonitoringSubscription` is missing.
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-cloudfront-monitoringsubscription.html
https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/viewing-cloudfront-metrics.html#enable-metrics

Looks like this resource has been added September 29, 2022.
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/ReleaseHistory.html

### Expected behavior

Resource type `AWS::CloudFront::MonitoringSubscription` should be accepted.

### Reproduction template

```yaml
AWSTemplateFormatVersion: 2010-09-09
Resources:
  CloudFrontDistributionMonitoringSubscription:
    Type: AWS::CloudFront::MonitoringSubscription
    Properties:
      DistributionId: !Ref CloudFrontDistribution
      MonitoringSubscription:
        RealtimeMetricsSubscriptionConfig:
          RealtimeMetricsSubscriptionStatus: Enabled
```

**Comment by hoonoh** [comment]
Sorry, but it seems there must have been some mixup with vscode extension or something.. after re-opening the vscode workspace cfn-lint started to accept this resource type.

**Comment by kddejong** [comment]
Good news @hoonoh thanks for closing the issue and let us know if you have any more issues.
